generated/nidcpower/nidcpower/session.py

```diff
@@ -2734,15 +2735,14 @@ def measure_multiple(self):
                 - **in_compliance** (bool) - Always None
 
         '''
-        import collections
-        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance'])
+        measurement = typing.NamedTuple('Measurement', [('voltage', 'float'), ('current', 'float'), ('in_compliance', 'bool')])
```

Comment:
FYI, these named tuples should be moved out of functions and into packages/modules, but there is a separate GitHub issue about that: https://github.com/ni/nimi-python/issues/1885